src/kernels/ipywidgets/ipyWidgetScriptSource.ts

```diff
@@ -86,17 +86,7 @@ export class IPyWidgetScriptSource {
                 this.uriConverter.resolveUri(response.request, response.response);
             }
         } else if (message === IPyWidgetMessages.IPyWidgets_Ready) {
-            // Send to UI (even if there's an error) instead of hanging while waiting for a response.
-            (this.scriptProvider ? this.scriptProvider?.getBaseUrl() : Promise.resolve())
```

Comment:
Nullables...
When the `scriptProvider` was null, we woudln't send this.
Hence fixed to send this baseUrl after the scriptProvider has been initialized as well.